debootstrap-ng.sh

```diff
@@ -288,7 +288,7 @@ prepare_partitions()
 	parttype[btrfs]=btrfs
 	# parttype[nfs] is empty
 
-	mkopts[ext4]='-q -m 2'
+	mkopts[ext4]='-O ^64bit,^metadata_csum,uninit_bg -q -m 2'
```

Comment:
@zador-blood-stained the only thing to test that I can try Docker, but I'm not sure if will not hit #318